tools/scripts/deno/generate-newsletters.js

```diff
@@ -1,5 +1,7 @@
 import { faker } from 'https://cdn.skypack.dev/@faker-js/faker@v7.6.0';
 
+const YEAR_IN_MS = 1000 * 60 * 60 * 24 * 365;
```

Comment:
Debatable whether this is a year in ms (leap years?) but for the purpose of generating newsletters it's close enough :-)